CHANGELOG.md

```diff
@@ -8,6 +8,31 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.8.0] - 2024-04-28
+
+### ⚠️ Notice ⚠️
+
+This release is the last to support Go `1.19`. The next release will require at least Go `1.20`.
```

Comment:
> The next release will require at least Go `1.20`

This is incorrect. Supposedly, the next release will require at least Go `1.21`.

Notice that in otelchi `v0.8.0` we are using otel `v1.24.0` which requires Go `1.20` already (see [here](https://github.com/open-telemetry/opentelemetry-go/blob/v1.24.0/go.mod#L3) for details).

If we check next version of otel `v1.24.0` which is `v1.25.0` we can see that it requires Go `1.21` (see [here](https://github.com/open-telemetry/opentelemetry-go/blob/v1.24.0/go.mod#L3) for details).